02-microarray/pathway-analysis_microarray_03_gsea.Rmd

```diff
@@ -333,15 +336,169 @@ Looks like we were able to successfully get rid of the duplicate gene identifier
 
 ## Perform gene set enrichment analysis (GSEA)
 
-_Addressed in upcoming PR_
+The goal of GSEA is to determine whether the members of a set of genes are randomly distributed throughout a pre-ranked gene list or primarily found at the top or bottom of the list meaning that they are highly- or lowly-ranked.
+As mentioned earlier, hallmark gene sets summarize and represent specific well-defined biological states or processes and display coherent expression of genes so we would expect the latter distribution. [@clusterProfiler-book].
```

Comment:
I don't think we _expect_ the latter distribution. It depends on the experiment! We might not always expect a particular pathway to be perturbed between our two groups. The way we run GSEA here (and almost every other way folks use GSEA) is a comparison between groups.

Reply:
Ahh okay that makes sense, will update this section to more accurately reflect what's going on here.